crates/experimentation_platform/src/api/experiments/types.rs

```diff
@@ -165,14 +163,34 @@ pub struct StatusTypes(
     pub  Vec<ExperimentStatusType>,
 );
 
+#[derive(Deserialize, Debug, Clone)]
+#[serde(rename_all = "snake_case")]
+pub enum ExperimentSortOn {
+    LastModifiedAt,
+    CreatedAt,
+}
+
+impl Default for ExperimentSortOn {
+    fn default() -> Self {
+        Self::LastModifiedAt
+    }
+}
+
 #[derive(Deserialize, Debug)]
 pub struct ExpListFilters {
     pub status: Option<StatusTypes>,
     pub from_date: Option<DateTime<Utc>>,
     pub to_date: Option<DateTime<Utc>>,
+    pub experiment_name: Option<String>,
+    pub experiment_ids: Option<StringArgs>,
+    pub created_by: Option<StringArgs>,
+    pub context: Option<String>,
+    pub sort_on: Option<ExperimentSortOn>,
+    pub sort_by: Option<SortBy>,
```

Comment:
do we want to have this as a separate filter ?
if so, then shouldn't we change it for `context/list `as well
there I had coupled `sort_on` and `sort_by` to a single enum list

Reply:
We should change context/list as well, but that is out of scope of this PR. Regarding this being a separate filter, I think for experiments, overrides and functions we will have different filters, all others should work with the common filter type

Reply:
Solved with PaginationParams